doc/reference/dune/include_subdirs.rst

```diff
@@ -35,3 +35,8 @@ following stanzas:
 - ``library``
 - ``executable(s)``
 - ``test(s)``
+
+
+.. warning::
+
+   As of now, there is an `issue <https://github.com/ocaml/dune/issues/11119>`_ when using ``(include_subdirs qualified)`` and lexing/parsing rules (e.g., ``ocamllex`` and ``menhir``).
```

Comment:
Sorry, but I don't understand what this sentence is trying to say. Do you mean that `(include_subdirs)` cannot be used in combination with `(ocamllex)` or `(menhir)`?

Reply:
For the qualified mode yes that is the case.

Reply:
Yes, cf. https://github.com/ocaml/dune/issues/11119